### PR TITLE
pep8 on printing.py

### PR DESCRIPTION
**Pull request by thomasmesnard**

### CODE REVIEW

File: theano/printing.py [hunk 1]
```diff
@@ -1288,13 +1295,15 @@ def hex_digest(x):
         try:
             import hashlib
         except ImportError:
-            raise RuntimeError("Can't run hex_digest because hashlib is not available.")
+            raise RuntimeError("Can't run hex_digest"
+                               "because hashlib is not available.")
     assert isinstance(x, np.ndarray)
-    rval = hashlib.md5(x.tostring()).hexdigest()
+    rval = hashlib.md5(x.tostring).hexdigest()
```
Comment (abergeron) [inline theano/printing.py]: This change is bad.  We need the parenthesis since this is a method we call.

**Comment by thomasmesnard** [comment]
It's done. Thank's a lot !

**Comment by abergeron** [comment]
There are still two flake8 errors according to travis.

**Comment by thomasmesnard** [comment]
Yes because it creates problems when converting Python 2 to Python 3. The problem may be fix now. I m waiting the result of Travis.

Thanks

**Comment by abergeron** [comment]
You can run `theano-nose test/test_flake8.py` from the theano source directory to check the result of that test yourself.

**Comment by thomasmesnard** [comment]
nosetests test_flake8.py give me no error. Pierre Luc and I think that when Travis convert printing.py in python3 with the function 2to3 it create new pep8 errors.

**Comment by abergeron** [comment]
The latest commit doesn't have errors on Travis.

**Comment by carriepl** [comment]
Looks fine. Thanks.

**Pull request merged**